data/authorities.json

```diff
@@ -183,6 +183,21 @@
       }
     }
   },
+  "DC": {
+    "city": {
+      "dc-dc-sustainable-energy-utility": {
+        "name": "DC Sustainable Energy Utility",
+        "city": "Washington, DC",
+        "county": "District of Columbia"
+      }
+    },
+    "state": {
+      "dc-dc-department-of-energy-and-environment": {
+        "name": "DC Department of Energy and Environment"
+      }
+    },
+    "utility": {}
```

Comment:
This will make it so that there are no utility options available in the selector, which I think will be confusing. (The options presented to users are the intersection of this set and the results from `zip_to_utility` in sqlite.) I know there are no incentives offered by the utility in DC, but we still ought to fix this.

The results from sqlite:
```
sqlite> select * from utilities where utility_id like 'dc-%';
utility_id                 name
-------------------------  ----------------------
dc-pepco                   Pepco
dc-washington-gas          Washington Gas
dc-potomac-electric-power  Potomac Electric Power
```

The only one you need to add here is Pepco. (Potomac Electric Power is its full legal name, but it's branded to customers as Pepco.)

You'll also need to follow the Utility Data section in `scripts/README.md` to modify the script that generates the utility data, to exclude Washington Gas and override Potomac Electric Power -> Pepco.